recipes/ssht/all/conanfile.py

```diff
@@ -0,0 +1,59 @@
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+from glob import glob
+import os
+
+
+
+class SshtConan(ConanFile):
+    name = "ssht"
+    license = "GPL-3.0-or-later"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/astro-informatics/ssht"
+    description = "Fast spin spherical harmonic transforms"
+    settings = "os", "arch", "compiler", "build_type"
+    topics = ("physics", "astrophysics", "radio interferometry")
+    options = {"fPIC": [True, False]}
+    default_options = {"fPIC": True}
+    requires = "fftw/3.3.9"
+    generators = "cmake", "cmake_find_package", "cmake_paths"
+    exports_sources = ["CMakeLists.txt"]
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.compiler == "Visual Studio":
+            raise ConanInvalidConfiguration("SSHT requires C99 support for complex numbers.")
+        del self.settings.compiler.cppstd
+        del self.settings.compiler.libcxx
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = glob('ssht-*/')[0]
+        os.rename(extracted_dir, self._source_subfolder)
+
+    @property
+    def cmake(self):
+        if not hasattr(self, "_cmake"):
+            self._cmake = CMake(self)
+            self._cmake.definitions["tests"] = False
+            self._cmake.definitions["python"] = False
+            self._cmake.configure(build_folder=self._build_subfolder)
+        return self._cmake
+
+    def build(self):
+        self.cmake.build()
+
+    def package(self):
+        self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
+        self.cmake.install()
+
+    def package_info(self):
```

Comment:
You'll need to set the `filenames` for https://github.com/astro-informatics/ssht/blob/main/cmake/SshtConfig.in.cmake to be generated correctly by conan

Reply:
I'm not sure I understand this comment. As far as I can tell, conan generates both xxxConfig.cmake and xxxTargets.cmake in the build directory. I've actually removed generating those upstream, since conan did not pick them up.